pkg/scalers/pulsar_scaler.go

```diff
@@ -178,13 +179,24 @@ func parsePulsarMetadata(config *ScalerConfig) (pulsarMetadata, error) {
 
 	meta.msgBacklogThreshold = defaultMsgBacklogThreshold
 
+	// FIXME: msgBacklog support DEPRECATED to be removed in v2.13
+	fmt.Println(config.TriggerMetadata)
 	if val, ok := config.TriggerMetadata[msgBacklogMetricName]; ok {
+		logger.V(1).Info("\"msgBacklog\" is deprecated and will be removed in v2.13, please use \"msgBacklogThreshold\" instead")
```

Comment:
v2.11 has already been released and this change will be included in v2.12, so the removal will be in 2.14
```suggestion
		logger.V(1).Info("\"msgBacklog\" is deprecated and will be removed in v2.14, please use \"msgBacklogThreshold\" instead")
```

Reply:
Thank @JorTurFer for the review. I will apply the same fix on the test case too!